### ISSUE TITLE
S3 web server functionality does not support IndexDocument or ErrorDocument

### ISSUE THREAD

**Issue by bb-cklimas** [localstack/localstack#1734]
Setting an index and error document for an S3 bucket in LocalStack seems to succeed, e.g. with aws-cli:

```
aws website s3://my-bucket/ --index-document index.html --error-document index.html
```

However, HTTP requests to the S3 bucket don't seem to honor these settings. I get a bucket listing instead of the index document, and a not found error that looks like it is generated by LocalStack instead of the error document.

┆Issue is synchronized with this [Jira Bug](https://localstack.atlassian.net/browse/LOC-136) by [Unito](https://www.unito.io/learn-more)

**Comment by whummer** [comment]
Addressed (partially) in #1780 - support for IndexDocument still missing

**Comment by macnev2013** [comment]
Thanks for reporting @bb-cklimas. This feature was added in [#3690](https://github.com/localstack/localstack/pull/3690)  - Can you please pull the `latest` version of docker image and try again. Thanks.
